examples/neural_networks/plot_mlp_training_curves.py

```diff
@@ -64,12 +70,19 @@ def plot_on_dataset(X, y, ax, name):
         print("training: %s" % label)
         mlp = MLPClassifier(verbose=0, random_state=0,
                             max_iter=max_iter, **param)
-        mlp.fit(X, y)
+      
```

Comment:
this line has trailing white spaces but that's not really important. LGTM anyway.

thanks @martinoywa